imageregistry/v1/types.go

```diff
@@ -195,6 +204,16 @@ type ImageRegistryConfigStorageS3 struct {
 	// Optional, defaults to false.
 	// +optional
 	VirtualHostedStyle bool `json:"virtualHostedStyle"`
+	// trustedCA is a reference to a ConfigMap containing a CA certificate
+	// bundle that should be trusted when establishing a connection to S3.
+	// It is used merged with the system default trust bundle.
```

Comment:
Copy and pasta this last line here ?

Reply:
It is not added to the system trust bundle.

Reply:
This CA bundle won't be used for other connections (at least be the operator, the registry is buggy on that side).

Reply:
Updated to
```
	// trustedCA is a reference to a config map containing a CA bundle. The
	// image registry and its operator use certificates from this bundle
	// together with certificates from the system trust bundle to verify S3
	// server certificates.
```